src/content/atlassian.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * @name Atlassian
```

Comment:
Maybe it's better to keep the Atlassian / Jira naming? So that people know that it's for Jira. 😅